### PR TITLE
init with google directions api calls

### PR DESCRIPTION
**Pull request by nlehuby**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+__pycache__/
+auth_params/
+test_cases/*.csv~
```
Comment (TeXitoi) [inline .gitignore]: you can just ignore `*~` directly.

File: connectors.py [hunk 2]
```diff
@@ -3,29 +3,23 @@
...
+    logger = logging.getLogger(__name__)
+    if mode not in ("walking", "bicycling", "driving"):
         logger.error("Le mode {} est inconnu - valeurs acceptées : walking, biking, driving".format(mode))
         return
 
     origin = "{},{}".format(from_tuple[0], from_tuple[1])
     destination = "{},{}".format(to_tuple[0], to_tuple[1])
 
-    url = "http://maps.googleapis.com/maps/api/directions/json?origin={}&destination={}&mode={}".format(origin, destination, mode)
+    url = "https://maps.googleapis.com/maps/api/directions/json?origin={}&destination={}&mode={}&key={}".format(origin, destination, mode, GOOGLE_TOKEN)
```
Comment (kinnou02) [inline connectors.py]: You should name your placeholder, it's easier when you have more than one

File: router_tester.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
+# encoding: utf-8
+
+import csv
+import connectors as router
+import logging
+
+def init_log():
+    logger = logging.getLogger("connectors")
```
Comment (kinnou02) [inline router_tester.py]: You should use [basicConfig](https://docs.python.org/2/library/logging.html#logging.basicConfig) it's easier and will setup the root logger

File: router_tester.py [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    logger.addHandler(steam_handler)
+    return logger
+
+def get_test_cases_from_csv_file(csv_file) :
+    with open(csv_file,'r') as f:
+        csv_reader = csv.DictReader(f, delimiter=';')
+        for row in csv_reader:
+            #test_case = {"to" : (45.77316,3.06115), "from": (45.77090,3.08228), "mode": "bicycling"}
+            test_case = {}
+            test_case["from"] = row['origin'].split('/')
```
Comment (kinnou02) [inline router_tester.py]: You store coordinate in multiple projection system?

File: connectors.py [hunk 5]
```diff
@@ -39,7 +33,6 @@ def get_distance_and_duration_from_google_directions(from_tuple, to_tuple, mode)
     return {'distance' : google_response['routes'][0]['legs'][0]['distance']['value'], 'duration': google_response['routes'][0]['legs'][0]['duration']['value']}
 
 if __name__ == '__main__':
```
Comment (kinnou02) [inline connectors.py]: This main doesn't seem useful anymore

**Pull request merged**